node/tests/webView2Test.ts

```diff
@@ -0,0 +1,41 @@
+import { test as base } from '@playwright/test';
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import childProcess from 'child_process';
+
+export const test = base.extend({
+  browser: async ({ playwright }, use, testInfo) => {
+    const cdpPort = 10000 + testInfo.workerIndex;
+    const executable = path.join(__dirname, '../../webview2-app/bin/Debug/net6.0-windows/webview2.exe');
+    fs.accessSync(executable, fs.constants.X_OK); // Make sure that the executable exists and is executable
+    const userDataDir = path.join(fs.realpathSync.native(os.tmpdir()), `playwright-webview2-tests/user-data-dir-${testInfo.workerIndex}`);
+    const webView2Process = childProcess.spawn(executable, [], {
+      shell: true,
+      env: {
+        ...process.env,
+        WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS: `--remote-debugging-port=${cdpPort}`,
+        WEBVIEW2_USER_DATA_FOLDER: userDataDir,
+      }
+    });
+    await new Promise<void>(resolve => webView2Process.stdout.on('data', data => {
+      if (data.toString().includes('WebView2 initialized'))
+        resolve();
+    }));
+    const browser = await playwright.chromium.connectOverCDP(`http://127.0.0.1:${cdpPort}`);
+    await use(browser);
+    await browser.close()
+    childProcess.execSync(`taskkill /pid ${webView2Process.pid} /T /F`);
+    fs.rmdirSync(userDataDir, { recursive: true });
```

Comment:
Will the directory delete be impacted by the lazy cleanup from webview2 app quit?

Reply:
Yup. e.g. Crashpad in Chromium does take forever to cleanup. We can pass some flags to prevent that from happening what we do inside Playwright: https://github.com/microsoft/playwright/blob/26150b7064754c8a8d9e5fe65b7a52e9919ee3eb/packages/playwright-core/src/server/chromium/chromium.ts#L328-L360